javascript-forms/form-values.js

```diff
@@ -0,0 +1,13 @@
+var $contactForm = document.querySelector('#contact-form');
+var messageData = {};
```

Comment:
Make sure you are creating this object inside of the function. You always want to use locally defined objects for situations like this.